pages/linux/sic.md

```diff
@@ -0,0 +1,29 @@
+# sic
+
+> simple irc client.
+> Part of suckless.org tools.
+> More information: <https://tools.suckless.org/sic/>.
+
+- Connect to host (default: irc.oftc.net) with nickname (default: $USER):
+
+`sic -n {{nickname}}`
+
+- Connect to a different host with PASS connection password:
+
+`sic -h {{host}} -n {{nickname}} -k {{password}}`
```

Comment:
Specifying a password on the command line like this is not very secure. Is there a way to prompt the user for a password, or read it from a file?

Reply:
@sbrl No. 
have a look [here](https://tools.suckless.org/sic/) =]

Reply:
This is unfortunate but it doesn't matter much.  It's their fault if the command is bad, we're just documenting how to use it.

Reply:
Exactly.